run

```diff
@@ -10,6 +10,10 @@ else
   rm /tmp/server-*.pem
 fi
 
+# This is a hack due to musl libc's DNS handling
+cat /etc/resolv.conf | head -n 3 > dns
+cat dns > /etc/resolv.conf
```

Comment:
Yeah, that's intended. If it's a cluster with a single master, than only the first DNS server is valid. I'll move it to another PR.